agr_java_core/src/main/java/org/alliancegenome/core/variant/converters/AlleleVariantSequenceConverter.java

```diff
@@ -118,7 +118,7 @@
                 }
 
             }
-
+            agrAllele.setPrimaryKey(variant.getPrimaryKey());
```

Comment:
@jt15  shouldn't this be done in the index? This is an issue in other places.

Reply:
sure we can do it in the api

Reply:
OH... I got mixed up... yes this is where this should be done... and you shouldn't need all that code in the AlleleVariantIndexService.java if the AlleleVariantSequenceConverter.java makes sure there is a primaryKey before indexing the allelel.

Reply:
Unless the LTP doesn't have one?

Reply:
do want id field to put it back in the AlleleVariantSequenceConverter.java and remove the lines from api code?

Reply:
ya sorry... its been a long day I got mixed up... if we get the primaryKey into the index than it will take care of multiple issue elsewhere

Reply:
yes

Reply:
its reverted now. added id to avsconverter and removed api code lines